dev/baked/Dockerfile

```diff
@@ -0,0 +1,47 @@
+# BUILD IMAGE --------------------------------------------------------
+ARG GO_VERSION=1.22
+FROM golang:${GO_VERSION}-bookworm as builder
+
+WORKDIR /app
+
+SHELL ["/bin/bash", "-c"]
+
+RUN apt-get update &&  \
+    apt-get install -y curl git jq && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
+
+RUN curl -L https://foundry.paradigm.xyz | bash && \
+    source ~/.bashrc && \
+    foundryup -v nightly-2044faec64f99a21f0e5f0094458a973612d0712 &&  \
```

Comment:
Can we pull the version out to a build arg to make it more visible and easier to maintain? Then we can have the version live in one place and use it everywhere